test/regression/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#![no_std]
+extern crate alloc;
+use alloc::{vec, vec::Vec};
+
+fn simple_linear_regression(values: &[(f64, f64)]) -> (f64, f64) {
```

Comment:
it's just the forward step, we still need the backward step.  I think its better to run https://docs.rs/linreg/latest/linreg/ directly